jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/explorer/CodeWhispererExplorerActionManager.kt

```diff
@@ -187,7 +187,7 @@ internal class CodeWhispererExploreActionState : BaseState() {
 }
 
 // TODO: Don't remove IsManualEnabled
```

Comment:
do we still need this TODO, since IsManualEnabled is already part of the enum class? May be you should consider adding some context here to indicate why someone should not remove IsManualEnabled and remove the `TODO:` prefix

Reply:
This is currently unused, but we need to make sure removing it is backward-compatiable with the existing schema for loading/saving states. This is worth to be addressed in a standalone PR.